content/fr/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -218,7 +218,7 @@ To start using your cluster, you need to run the following as a regular user:
 
 You should now deploy a pod network to the cluster.
 Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
-  https://kubernetes.io/docs/concepts/cluster-administration/addons/
+  /docs/concepts/cluster-administration/addons/
```

Comment:
je pense qu'un lien est souhaitable ici.

Reply:
yeah, you are right!
![image](https://user-images.githubusercontent.com/1580956/54828382-6f6b1f80-4c8a-11e9-976d-7bbea1503a7d.png)

Reply:
This should be full link for sure.

Reply:
The problem is if we put back the full link, we will see the English content instead of French content. We need to think about this.

Reply:
so it might be:
https://kubernetes.io/fr/docs/concepts/cluster-administration/addons/

Reply:
Yes you are right. Once this page will be translated it will show up in French :)

Reply:
cool, I will do the update shortly.